tutorials/tmva/TMVA_SOFIE_RSofieReader.C

```diff
@@ -0,0 +1,67 @@
+/// \file
+/// \ingroup tutorial_tmva
+/// \notebook -nodraw
+/// This macro provides an example of using a trained model with Keras
+/// and make inference using SOFIE with the RSofieReader class
+/// This macro uses as input a Keras model generated with the
+/// TMVA_Higgs_Classification.C tutorial
+/// You need to run that macro before to generate the trained Keras model
+///
+///
+/// Execute in this order:
+/// ```
+/// root TMVA_Higgs_Classification.C
+/// root TMVA_SOFIE_RSofieReader.C
+/// ```
+///
+/// \macro_code
+/// \macro_output
+/// \author Lorenzo Moneta
+
+using namespace TMVA::Experimental;
+
+
+using namespace TMVA::Experimental;
```

Comment:
This line is repeated. It's also there in line 24.

Reply:
Thank you! I will fix this!